vendor/vendor.json

```diff
@@ -251,10 +251,10 @@
 			"revisionTime": "2017-07-27T06:48:18Z"
 		},
 		{
-			"checksumSHA1": "ANR/7MdWTtjTzxFZSuKE0Ss5j0s=",
```

Comment:
Let's break this change out into it's own Pull Request. We shouldn't bump library versions with new features unless they are dependent on one another and even then, two PRs is preferred.

Reply:
Please don't bump `cloudflare-go` in PRs. If needed, just let us know and we will commit updated upstream version.

Reply:
Restored from master